docs/sphinx/sst/running_simeng_with_sst.rst

```diff
@@ -0,0 +1,361 @@
+main contact during simulation. The ``CacheController`` class is aliased as **Cache** in SST. Caches in SST communicate through ports ``low_network_0``  and 
+``high_network_0``. The ``high_network_0`` port is used to connect to either the SimEng core (via ``StandardMem`` interface) or to higher levels of cache, 
+whereas the ``low_network_0`` port is used to connect to lower levels of cache or the main memory of the SST memory model. 
+
+The Cache component also has some subcomponent slots (``memlink`` and ``cpulink``) which can be used instead of ports to establish communication as well.
+
+Information about different configuration parameters, ports and subcomponent slots required for simulation can be found using the command: ``sst-info memHierarchy.Cache`` 
+
+MemoryController
+****************
+The ``MemoryController`` class implements the memory controller interface. The memory controller object sits between the last level cache and the memory backend 
+(including the converter), and it serves as an entry point for accessing the main memory. The memory controller object is derived from the ``Component`` class, 
+meaning that it can be instantiated in the configuration file, and connected to the upper-level components. All memory requests to the main memory (e.g RAM) go 
+through the ``MemoryController``.
+
+The controller object also serves as an upper-level container for other memory-related components. These related components are loaded into subcomponent slots. 
+Its ``backend`` slot stores a reference to the memory backend that implements the timing of the main memory. Its ``cpulink`` slot stores a reference to the 
+memory link object (either a direct link or a network-on-chip endpoint) that the controller uses to communicate with upper-level components in the memory 
+hierarchy. However, instead of using the ``cpulink`` slot, the controller can also communicate with higher memory components through the ``direct_link`` port.
+
+Information about the different configuration parameters, ports and subcomponent slots required for simulation can be found using the command: ``sst-info memHierarchy.MemController``
+
+MemoryBackend
+*************
+The memory backend implements the timing model of the main memory. All memory backend implementations must inherit from the base class, ``MemBackend``, present in 
+the file ``memBackend.h`` in the **SST-Elements** source code. Class ``MemBackend`` is an abstract class and hence cannot be directly instantiated. The class is 
+also a child class of ``SubComponent``, and it must be loaded into the slot of the memory controller, as we have already seen earlier. SST provides a variety of 
+memory backends to choose from e.g. ``simpleMem``, ``simpleDRAM``, etc (These can be specified in the configuration params for ``MemoryController``). The memory 
```

Comment:
*"memory backends to choose from e.g. ``simpleMem``, ``simpleDRAM``, etc. These can be specified in the configuration params for ``MemoryController``."